native/src/hooks/useLoadTuNewsElement.ts

```diff
@@ -0,0 +1,23 @@
+import { useCallback } from 'react'
+
+import { createTunewsElementEndpoint, ReturnType, TunewsModel, useLoadAsync } from 'api-client'
+
+import { tunewsApiUrl } from '../constants/endpoint'
+
+type UseLoadTuNewsElementProps = {
+  newsId: string
+}
+
+const useLoadTuNewsElement = ({ newsId }: UseLoadTuNewsElementProps): ReturnType<TunewsModel[]> => {
```

Comment:
Why do you use an Array as return type? It is just one element?

Reply:
True, was necessary before bc both tunews list and tunews detail was handled in one component. Will change this! Good suggestion :pray: